src/System.Globalization/tests/NumberFormatInfo/NumberFormatInfoNumberNegativePattern.cs

```diff
@@ -2,74 +2,44 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using System;
-using System.Globalization;
+using System.Collections.Generic;
 using Xunit;
 
 namespace System.Globalization.Tests
 {
     public class NumberFormatInfoNumberNegativePattern
     {
-        // TestDefault: Verify default value of property NumberNegativePattern
-        [Fact]
-        public void TestDefault()
+        public static IEnumerable<object[]> NumberNegativePattern_TestData()
         {
-            NumberFormatInfo nfi = new NumberFormatInfo();
-            int expected = nfi.NumberNegativePattern;
-            Assert.Equal(1, expected);
+            yield return new object[] { NumberFormatInfo.InvariantInfo, 1 };
+            yield return new object[] { new CultureInfo("en-US").NumberFormat, 1 };
         }
 
-        // TestSetValue: Verify set value of property NumberNegativePattern
-        [Fact]
-        public void TestSetValue()
+        [Theory]
+        [MemberData(nameof(NumberNegativePattern_TestData))]
+        public void NumberNegativePattern(NumberFormatInfo format, int expected)
```

Comment:
Shouldn't this be named "NumberNegativePattern_Get" or something like that to mirror the set version below?